server/db/db_upgrade.py

```
@@ -547,94 +547,295 @@ def ensure_plugins_tables(sql) -> bool:
+
+    # Drop views first — ALTER TABLE RENAME COLUMN will fail if a view
+    # references the old column name and the view SQL cannot be rewritten.
+    for view_name in ("Events_Devices", "LatestEventsPerMAC", "Sessions_Devices",
+                      "Convert_Events_to_Sessions", "LatestDeviceScan", "DevicesView"):
+        sql.execute(f"DROP VIEW IF EXISTS {view_name};")
+
+    renamed_count = 0
+
+    for table, column_map in _CAMELCASE_COLUMN_MAP.items():
+        # Check table exists
+        sql.execute(f"SELECT name FROM sqlite_master WHERE type='table' AND name=?", (table,))
```

Ruff (0.15.5)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)